requirements.txt

```diff
@@ -0,0 +1,42 @@
+# SPDX-License-Identifier: MIT OR Apache-2.0
+# SPDX-FileCopyrightText: The Ferrocene Developers
+
+# This file was autogenerated by uv via the following command:
+#    uv pip compile - -o requirements.txt
```

Comment:
Just wondering, where do your input requirements come from?

Reply:
This was done by following the instructions from https://github.com/astral-sh/uv?tab=readme-ov-file#getting-started

![image](https://github.com/ferrocene/ferrocene/assets/130903/ae3dd51b-f95a-45c8-89d0-d50b6c6f3232)

Is there a better way to approach this? (I've hardly touched python in over a decade)

Reply:
Generally it's best practice to declare your dependencies and some constraints  in a `requirements.in` file then compile those to a `requirements.txt` so you have the input requirements checked in to your repository.